File: cclapi/Models/Inventario.cs
namespace cclapi.Models
{
    public class Inventario(int Id, int Id_producto, int Cantidad)
    {
        public int Id { get; set; } = Id;
        public required int Id_producto { get; set; } = Id_producto;
        public required int Cantidad { get; set; } = Cantidad;
        public DateTime Last_updated { get; set; } = DateTime.Now;
    }
}

File: mobileapp/inventario/lib/models/inventario.dart
class Inventario {
  int id;
  int idProducto;
  int cantidad;
  DateTime lastUpdated = DateTime.now();

  Inventario(this.id, this.idProducto, this.cantidad) {
    lastUpdated = DateTime.now();
  }

  factory Inventario.fromJson(Map<String, dynamic> json) {
    return Inventario(
      json['id'],
      json['id_producto'],
      json['cantidad'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'id_producto': idProducto,
      'cantidad': cantidad,
    };
  }
}
